### DISCUSSION TITLE
It kept bugging at the fb_token check in check-repo :(

### DISCUSSION THREAD

**Discussion by neopagn** [fearocanity/ebtrfio-template#24] [Q&A] [answered]
![image](https://github.com/fearocanity/ebtrfio-template/assets/139479532/7a0e6b56-0652-4d4d-83ad-c971ea59224a)
i have no idea how the job keep bugging, i tried changing the page names, change the access token and stuff, but it is still error

**Comment by fearocanity** [comment, marked as answer]
Token is the problem.
The checker checks your token if its working and its the same as your page name, otherwise it will return error..

Your token is probably a user level token, not a page token.
Make sure to always set the `User or Page` dropdown to your Page, no matter what. You can repeat the step again from readme.

**Reply by neopagn** [reply]
Thank you so much for the reply, it was indeed the User level token that caused the bug, I didn't pay attention to that User or Page drop down. anw really awesome code <3, thanks again

**Reply by fearocanity** [reply]
Btw is your repo private? If yes (without self-hosted), you need to set it as Public because they will limit you if you hit their quota (while using runners in private)... Don't worry GitHub Secrets is secure.

**Reply by neopagn** [reply]
oh thanks for the heads up, i'll change it to public <3
